Makefile

```diff
@@ -0,0 +1,163 @@
+# Are your paths correct?
+#
+# Makefile expects to run at the project root, and not nested inside anywhere
+#
+#  cd $DESK/xvm; make
+
+# Use bash for recipes
+SHELL := /bin/bash
+
+# Keep partial builds but not partial recipes
+.NOTINTERMEDIATE:	
+
```

Comment:
This makefile may belong more in the javatools_backend directory than in the project root. We don't want to mistaken for a part of the global build system.